analyses/mutational-signatures/run_mutational_signatures.sh

```diff
@@ -9,28 +9,35 @@ set -o pipefail
 # Set the working directory to the directory of this file
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
+# Only fit for CNS signatures, which we'll do by default
+# This does require us to split the MAF files by strategy, but that's all
+CNS_FIT_ONLY=${OPENPBTA_CNS_FIT_ONLY:-1}
+
 # In CI we'll run an abbreviated version of the de novo signatures extraction
 ABBREVIATED_MUTSIGS=${OPENPBTA_QUICK_MUTSIGS:-0}
 
-
-# Run the mutational signatures analysis using existing signatures
-Rscript -e "rmarkdown::render('01-known_signatures.Rmd', clean = TRUE)"
-
 # Split up the consensus MAF files by experimental strategy (writes to scratch)
 Rscript --vanilla 02-split_experimental_strategy.R
 
-# Run the shell script for determining the number of signatures to use
-# with a low number of iterations if run in CI
-# argument 0 --> run denovo goodness of fit
-QUICK_MUTSIGS=$ABBREVIATED_MUTSIGS ANALYSIS=0 bash 03-de_novo_range_of_nsignatures.sh
+if [ "$CNS_FIT_ONLY" -lt "1" ]; then
```

Comment:
I don't know, but I find this easier to follow the logic of. 
```suggestion
if [ "$CNS_FIT_ONLY" == "0" ]; then
```